### ISSUE TITLE
RX 580

### ISSUE THREAD

**Issue by RaspberyPiProjectsandMore** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#101]
Hello, 
         Could I use this EFI Folder with a RX 580 instead of a RX 5700 RT?

**Comment by addisonduda** [comment]
Yep! I have been using it no problem since it was first published. Just remove SSDT-RX 5700 XT_Version 1.0.aml and replace it with SSDT-RX 580-Version 1.0.aml, in both the plist file and the APCI folder. Let me know if you need any help!

**Comment by wayoo** [comment]
@addisonduda  hi, where can i download `SSDT-RX 580-Version 1.0.aml` file?

**Comment by RaspberyPiProjectsandMore** [comment]
@addisonduda Thanks so much! I'm about to buy the parts to build a hackintosh so I wanted to make sure if I could use the rx 580 instead. I'll be trying it out soon. Thanks for your help!

**Comment by addisonduda** [comment]
Just following up that the modification I mentioned earlier is no longer needed. The current EFI is more general and provides the same, if not better GPU scores in benchmarks.

**Comment by RaspberyPiProjectsandMore** [comment]
Oh awesome! Thanks for your help!
